### DISCUSSION TITLE
Installation - commande line

### DISCUSSION THREAD

**Discussion by b4b857f6ee** [Security-Onion-Solutions/securityonion#7616] [Unsupported Versions] [answered]
Hello,

Do we have any solution to install in command line without select X or Y option ?
Like use a configuration file/parameter and run the all installation in one command? :)

Thank you.

**Comment by dougburks** [comment, marked as answer]
Our Professional Services team can help with automation. Please see https://securityonionsolutions.com/support for more information and to reach out to them.

**Reply by b4b857f6ee** [reply]
Ok thank you.
My goal is to speedup the deployement to use it for student. So they don't have to wait and to configure it.
I will not have this problem but the update IP process looks like, this is not 100% accurate ^^.
I already do it for the agent deployement to help them to focus on detection.
